logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/ScheduleNewAssetMessageUseCase.kt

```diff
@@ -204,9 +204,7 @@ internal class ScheduleNewAssetMessageUseCaseImpl(
         message: Message,
         conversationId: ConversationId
     ): Either<CoreFailure, Unit> =
-        messageSender.sendPendingMessage(conversationId, message.id).onFailure {
-            kaliumLogger.e("There was an error when trying to send the asset on the conversation")
-        }
+        messageSender.sendPendingMessage(conversationId, message.id)
```

Comment:
Why are we trimming these error messages? I thought we wanted to be more verbose when it comes to logging errors?

Reply:
Nevermind, I just saw there was some generic error logging added in the underlying layers of the `MessageSender`